### PR TITLE
[Improve] [core] Using right exit code instead of -1

### PR DESCRIPTION
**Pull request by asdf2014**
## Purpose of this pull request

> On POSIX systems the standard exit code is 0 for success and any number from 1 to 255 for anything else.

This PR uses right exit code instead of -1 for the shell script.

## Check list

* [x] Code changed are covered with tests, or it does not need tests for reason:
* [x] If any new Jar binary package adding in you PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/developement/NewLicenseGuide.md)
* [x] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs

### CODE REVIEW

File: seatunnel-core/seatunnel-core-flink/src/main/bin/start-seatunnel-flink.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ fi
 is_exist() {
     if [ -z $1 ]; then
       usage
-      exit -1
+      exit 1
```
Comment (ruanwenjun) [inline seatunnel-core/seatunnel-core-flink/src/main/bin/start-seatunnel-flink.sh:38]: Maybe exit 0 is better? It seems `exist 1` means error return.

Comment (asdf2014) [reply seatunnel-core/seatunnel-core-flink/src/main/bin/start-seatunnel-flink.sh:38]: @ruanwenjun No, 1 is better. It reminds users this requires option. And the original -1 also means error, which is same as 1.

Comment (ruanwenjun) [reply seatunnel-core/seatunnel-core-flink/src/main/bin/start-seatunnel-flink.sh:38]: OK, it makes sense to me.

**Review by ruanwenjun — APPROVED** [review]
LGTM

**Review by davidzollo — APPROVED** [review]
LGTM

**Pull request merged**